src/pages/workspaces/actions.js

```diff
@@ -0,0 +1,10 @@
+import { Rawls } from 'src/libs/ajax'
+
+
+export const loadWorkspaces = () => (dispatch) => {
```

Comment:
A note on code organization. Actions can be provided inline to `connect`, or factored out into a separate file like this. I find it's nice to pull out 'thunk' (async) actions like this one, at the very least.

Reply:
They tend to be bulky, and not as closely related to the components they're called from. Moving  them into their own space keeps the components more streamlined. It can also open up some opportunities to share code between various async functions.